Description: Don't print version info on help output
 The help message had a placeholder which is supposed to be filled in by
 the normal bash build script.
 .
 gocryptfs (1.0-1) UNRELEASED; urgency=low
 .
   * Initial release (Closes: TODO)
Author: David Steele <steele@debian.org>


--- gocryptfs-1.0.orig/main.go
+++ gocryptfs-1.0/main.go
@@ -161,8 +161,8 @@ func printVersion() {
 		humanTime = fmt.Sprintf("%d-%02d-%02d", t.Year(), t.Month(), t.Day())
 	}
 	built := fmt.Sprintf("%s %s", humanTime, runtime.Version())
-	fmt.Printf("%s %s; go-fuse %s; %s\n",
-		tlog.ProgramName, GitVersion, GitVersionFuse, built)
+	fmt.Printf("%s\n",
+		tlog.ProgramName)
 }
 
 func main() {
